docs/CONTRIBUTING.md

```diff
@@ -1,38 +1,136 @@
+|---------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Participate in the community                      | What makes this tool successful is its international & diverse community. One of the best ways to contribute is simply to be part of it!  Participate in the discussions, answer on the newly opened issues, help others, attend the community meetings. This is all happening on the Slack channel #gtfs-validators.                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Comment or vote on issues issues already opened.  | We use the [GitHub issue tracker](https://github.com/MobilityData/gtfs-validator/issues) to capture bugs, new feature ideas and improvements.  If you see an issue that is relevant to you, give a thumbs up or comment. This will increase the priority of the issue, and it will be solved earlier as a result!                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
```suggestion
| Comment or vote on issues issues already opened.  | We use the [GitHub issue tracker](https://github.com/MobilityData/gtfs-validator/issues) to capture bugs, new feature ideas and improvements. If you see an issue that is relevant to you, give a thumbs up or comment. This will increase the priority of the issue, and it will be solved earlier as a result!                                                                                                                                                                                                                                                                                                                                                                                                                     |
```